### PR TITLE
feat: add solution for 2024-09 day 26

### PR DESCRIPTION
**Pull request by stephentth**
solution for 2024-09 day 26, feedback is welcomed

### CODE REVIEW

File: daily-challenges/2024-09/26/stt20240926.spec.ts
Comment (minhphong306) [inline daily-challenges/2024-09/26/stt20240926.spec.ts:1]: Vui lòng rename file theo convention:
github_username.spec.ts

File: daily-challenges/2024-09/26/stt20240926.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const totalRow = await rows.count();
+
+    let visibleCount = 0;
+    for (let i = 0; i < totalRow; i++) {
+        const row = rows.nth(i);
+        if (await row.isVisible()) {
+            const classCell = row.locator('td').nth(2);
+            const classCellText = await classCell.innerText();
+            const hasSuffixA4 = classCellText.endsWith('A4');
+            expect(hasSuffixA4).toBe(true);
```
Comment (minhphong306) [inline daily-challenges/2024-09/26/stt20240926.spec.ts:29]: Nên sử dụng web-first assertion ở đây:
```suggestion
            const classCell = row.locator('td').nth(2);
            await expect(classCell).toContainText('A4');
```

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix comments.
Good work!!!

File: daily-challenges/2024-09/26/stt20240926.spec.ts (hunk 1 above)
Comment (stephentth) [reply daily-challenges/2024-09/26/stt20240926.spec.ts]: Thanks bạn

Comment (stephentth) [reply daily-challenges/2024-09/26/stt20240926.spec.ts]: Pls help to review again @minhphong306

**Pull request merged**